lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java

```diff
@@ -141,7 +141,14 @@ private CCAssignmentItem getContents(String assignmentRef) {
         switch (typeOfGrade) {
             case SCORE_GRADE_TYPE:
                 Integer scaleFactor = assignment.getScaleFactor() != null ? assignment.getScaleFactor() : assignmentService.getScaleFactor();
-                double maxPoints = new Double(assignmentService.getMaxPointGradeDisplay(scaleFactor, assignment.getMaxGradePoint()));
+                String aux = assignmentService.getMaxPointGradeDisplay(scaleFactor, assignment.getMaxGradePoint());
+                String decSeparator = formattedText.getDecimalSeparator();
+                double maxPoints = 0.0;
+                try {
+                    maxPoints = Double.valueOf(StringUtils.replace(aux, decSeparator, ".")) * 10;
```

Comment:
maxPoints should be multiplied by 10. On 12.x and 19.x is on that way.

Reply:
this seems wrong. the scale factor doesn't have to be 10, right @ern?

Reply:
What should we do then? Should I remove * 10 and create another jira to investigate why sacleFactor is 100?

Reply:
There should be no reason to multiply this by 10 here

Reply:
I have deleted * 10